resources/metadata/mtaBuild.yaml

```diff
@@ -227,6 +227,15 @@ spec:
         resourceRef:
           - name: commonPipelineEnvironment
             param: custom/buildSettingsInfo
+      - name: createBOM
+        type: bool
+        description: Creates the bill of materials (BOM) using CycloneDX plugin.
```

Comment:
Does it really use CycloneDX?

Reply:
Underneath yes.

Reply:
Should I just change it to: "Creates the bill of materials (BOM) using MBT tool" ? even MBT tool runs the cyclone DX plugin and combines the results ;)

Reply:
No, then it's fine, I just thought this might be a copy&paste issue.